prime-router/src/main/kotlin/azure/DatabaseAccess.kt

```diff
@@ -505,6 +507,25 @@ class DatabaseAccess(private val create: DSLContext) : Logging {
             ?.getValue(DSL.max(SETTING.VERSION)) ?: -1
     }
 
+    fun insertJti(jti: String, expiresAt: OffsetDateTime? = null, txn: DataAccessTransaction) {
```

Comment:
The JTI Cache is used to check for a replay attack, wherein the malicious party gets the token request, and attempts to send it again.

Reply:
We're using the database to spread this across the cluster, correct? This is persisted and the transaction committed before the party gets the token request, so if they replay on another instance the JTI will have been cached, correct?

Reply:
I think my calling this a "cache" was a bad idea.   Its a database store.  Yes, the database is currently universal in ReportStream, and the row is inserted in a transaction, before a response is sent back to the sender.